app/lib/efile/nj/nj_flat_eitc_eligibility.rb

```diff
@@ -57,11 +57,11 @@ def all_outside_fed_eitc_age_range?(*ages)
         end
 
         def is_under_income_total_limit?(intake)
-          if intake.filing_status_mfj?
-            intake.direct_file_data.fed_income_total < 25_511
-          else
-            intake.direct_file_data.fed_income_total < 18_591
-          end
+          intake.direct_file_data.fed_agi < if intake.filing_status_mfj?
```

Comment:
[dust] Assigning the if clause to a `threshold` variable here would make this more readable

Reply:
agree!

Reply:
Done, appreciate the suggestion!